### DISCUSSION TITLE
Codeigniter website working on localhost but not on live server

### DISCUSSION THREAD

**Discussion by Nick201986** [bcit-ci/CodeIgniter#6223] [General]
I have a CodeIgniter website that works perfectly on my local. But when I try to put it on the live Godaddy server it is not working getting the error "This page isn’t working [www.nulearn.in](http://www.nulearn.in/) is currently unable to handle this request. HTTP ERROR 500". I create a test page that is not linked with the MySQL database. That is working fine.

My .htaccess looks like this if it has something to do with it.

RewriteEngine On
RewriteCond %{HTTPS} !=on
RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301,NE]
RewriteCond $1 !^(index\.php|resources|robots\.txt)
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^(.*)$ index.php/$1 [L,QSA]

**Comment by Oscabrera** [comment]
topics related to.  https://github.com/bcit-ci/CodeIgniter/discussions/6222#discussion-5325555
